cli/src/main/java/com/devonfw/tools/ide/merge/xmlmerger/XmlMergeStrategy.java

```
@@ -60,13 +61,13 @@ protected void doMerge(Element templateElement, Element resultElement, ElementMa
    * @param matcher the {@link ElementMatcher}.
    */
   public void merge(Element templateElement, Element resultElement, ElementMatcher matcher) {
-
+    Logger logger = Logger.getLogger(XmlMergeStrategy.class.getName());
     try {
       doMerge(templateElement, resultElement, matcher);
     } catch (XmlMergeException e) {
-      throw e;
+      logger.warning("XML Merge Exception: " + e.getMessage());
     } catch (RuntimeException e) {
-      throw new XmlMergeException("Merge strategy " + this + " failed on " + XmlMergeSupport.getXPath(templateElement, true), e);
+      logger.warning("Merge strategy " + this + " failed on " + XmlMergeSupport.getXPath(templateElement, true) + ": " + e.getMessage());
```

Could you revert this changes as this is causing that all other errors are now also just logged and stacktraces of real errors are lost so we cannot support users hitting new problems and say exactly what went wrong (see e.g. #1038).

---

thanks, I deleted it.